tests/pools/abstract/abstract-pool.test.js

```diff
@@ -1,5 +1,6 @@
 const expect = require('expect')
 const { FixedThreadPool } = require('../../../lib/index')
+const { FixedClusterPool } = require('../../../lib/index')
```

Comment:
I think you can merge this to `const { FixedClusterPool, FixedThreadPool } = require('../../../lib/index')`